### PR TITLE
New NZGD data map

### PR DESCRIPTION
**Pull request by AndrewRidden-Harper**
The new NZGD data map is based on @lispandfound 's [template](https://github.com/ucgmsim/webplate) and is temporarily hosted at http://hypocentre:5000/ if you want to try it

### CODE REVIEW

**Review by sungeunbae — APPROVED** [review]

File: app/views.py [hunk 1]
```diff
@@ -1,121 +1,371 @@
...
+    soil_types_as_str = []
+    for soil_type in spt_df["Soil Type"]:
+
+        if len(soil_type) > 0:
+            combined_soil_str = ""
+            for soil_str in soil_type:
+                combined_soil_str += soil_str + " + "
+
+            ## Remove the last "and" from the string
+            combined_soil_str = combined_soil_str.strip(" + ")
```
Comment (lispandfound) [inline app/views.py]:
```suggestion
            combined_soil_str = " + ".join(soil_type)
```

File: app/templates/base.html [hunk 2]
```diff
@@ -10,6 +10,16 @@
...
     <!-- Load Plotly -->
     <script src="{{ url_for('static', filename='plotly.min.js.gz') }}"></script>
+
+    <!-- Use MathJax  -->
+    <script src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-mml-chtml.js"></script>
+    <script>
+        window.addEventListener('load', function() {
+            MathJax.typeset();
+        });
+    </script>
```
Comment (lispandfound) [inline app/templates/base.html]: This is a very big JS payload. You only use it on `index.html`. Consider moving this to the scripts block inside `index.html`.

That would be removing the above code and adding the following to index.html.
```
{% block scripts %}
    <script src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-mml-chtml.js"></script>
    <script>
        window.addEventListener('load', function() {
            MathJax.typeset();
        });
    </script>
{% endblock %}
```

**Review by lispandfound — APPROVED** [review]

**Review by lispandfound — CHANGES_REQUESTED** [review]
See the requested changes.

**Review by lispandfound — APPROVED** [review]

**Comment by joelridden** [comment]
Maybe when the Vs30 values are nan in the table to remove them e.g. (boore_2004 less than 10 meters)

**Review by joelridden — COMMENTED** [review]
Maybe when the Vs30 values are nan in the table to remove them e.g. (boore_2004 less than 10 meters)

**Comment by AndrewRidden-Harper** [comment]
Now it only shows Vs30 results for Vs to Vs30 correlations that are valid given the CPT's maximum depth. For the map display, if the selected Vs to Vs30 correlation does not allow a Vs30 value to be calculated, an explanation is shown in the map marker's hover text

**Review by joelridden — APPROVED** [review]

**Pull request merged**